src/main/java/de/tum/in/test/api/structural/StructuralTestProvider.java

```diff
@@ -78,6 +78,7 @@
 	protected static final String JSON_PROPERTY_TYPE = "type";
 	protected static final String JSON_PROPERTY_RETURN_TYPE = "returnType";
 	protected static final String JSON_PROPERTY_ENUM_VALUES = "enumValues";
+	protected static final String JSON_PROPERTY_Strict_Order = "strictOrder";
```

Comment:
```suggestion
	protected static final String JSON_PROPERTY_STRICT_ORDER = "strictOrder";
```
I would suggest we stick to the all upper-case naming of constants here as well.